src/css/gitbook.scss

```diff
@@ -0,0 +1,39 @@
+$gitbook-blue-grey: #f5f7f9;
+$gitbook-border-grey: #d4dadf;
+$gitbook-border-light: #e6ecf1;
+$gitbook-blue: rgb(56, 132, 254);
+
+html.gitbook {
+  #desktop-header {
+    background: white;
+    border-bottom: 1px solid $gitbook-border-grey;
+    box-shadow: 0 3px 8px 0 rgba(116, 129, 141, 0.1);
+  }
+
+  .bg-light {
+    background-color: $gitbook-blue-grey !important;
+  }
+
+  .desktop-nav {
+    border-right: 1px solid $gitbook-border-light;
+
+    .course-nav-list-item.active {
+      background-color: white;
+      border: 1px solid $gitbook-border-light;
+      border-right: none;
+
+      a {
```

Comment:
This rule causes all links under a list item to be colored blue, regardless of hierarchy tier:

![image](https://user-images.githubusercontent.com/12089658/76119335-3623e080-5fbd-11ea-8f73-302e3b9b6a4c.png)

I think maybe the text should only be blue on the first (selected) link.  I was thinking you could add an ID only to child links like this in `nav_item.html`:
```
<a class="text-uppercase text-dark{{ if $hasParent }} course-nav-link-child{{ end }}{{ if $isMenuCurrent }} active{{ end }}"
```

Then in `gitbook.scss`, you could change the nested a selector to look like this:

```
      a:not(.course-nav-link-child) {
        color: $gitbook-blue !important;
      }
```

Reply:
Also, just noticing that it's kind of a little odd how the child items wrap text there.

Reply:
should have made it more clear! ferdi shared some feedback that I'm sitll working on, so things aren't quite final yet

Reply:
Gotcha, I wasn't sure if you wanted a review or not yet but figured it couldn't hurt to give it a look.